netplan/cli/commands/get.py

```diff
@@ -41,27 +42,37 @@ def run(self):
         self.parse_args()
         self.run_command()
 
+    def dump_state(self, key, np_state, output_file):
+        if key == 'all':
+            np_state.dump_yaml(output_file=output_file)
+            return
+
+        if not key.startswith('network'):
```

Comment:
Hmm... I remember having a reason for doing that in the specific way it was done before; I can remember the details, though... Would you mind keeping it at this, to not change behaviour?
```suggestion
        if not key.startswith('network.') and not key == 'network':
```

I think one problem could be that if somebody tries to `netplan get networkINVALID`, this should return an error/empty tree. OTOH.. I think that might actually be handled by the `dump_yaml_subtree` function, as it won't find the `networkINVALID` prefix.

Reply:
One reason would be the handling of documents with more than one top-level key, but since we go through the entire parsing process, those would get tossed out anyway.

Unless you give me an actual case where the new code is problematic, I'd rather keep it as is, as I've noticed that weird code with unclear rationales tend to calcify.

I've added a test case for the `networkINVALID` query, though :)

Reply:
That's fair, I guess! All our tests are passing. Should we hit a regression, we'll fix it up and add a new test case for it.